example-apps/session-auth/index.js

```diff
@@ -30,7 +20,7 @@ const App = {
 
   beforeRequest: [includeSessionKeyHeader],
 
-  afterResponse: [sessionRefreshIf401],
```

Comment:
There was never a need for this as we already used to always call `throwForStaleAuth` for session auth.